guides/source/command_line.md

```diff
@@ -92,22 +92,28 @@ Rails will set you up with what seems like a huge amount of stuff for such a tin
 
 If you wish to skip some files or components from being generated, you can append the following arguments to your `rails new` command:
```

Comment:
The purpose of this section is to list integrations that can be opted-out of, rather than list all `--skip-*` CLI options.  Therefore...